### PR TITLE
K8SPG-159 remove old mongo-driver versions

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: go.mod [hunk 1]
```diff
@@ -1,26 +1,52 @@
 module github.com/Percona-Lab/percona-version-service
 
-go 1.14
+go 1.15
```
Comment (hors) [inline go.mod:3]: I think we need to use 1.17

**Review by hors — CHANGES_REQUESTED** [review]

**Review by delgod — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by egegunes — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Pull request merged**